akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -395,10 +396,22 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef, joinConfigCompatCh
       context.actorOf(propsWithDispatcher, name = "downingProvider")
     }
 
-    if (seedNodes.isEmpty)
-      logInfo("No seed-nodes configured, manual cluster join required")
-    else
+    if (seedNodes.isEmpty) {
+      if (isClusterBootstrapUsed)
+        logDebug("Cluster Bootstrap is used for joining")
+      else
+        logInfo("No seed-nodes configured, manual cluster join required")
```

Comment:
Maybe someone who has written manual cluster join code would also want to suppress this message. I think the message is most useful for someone who hasn't realized they need to think about how to form a cluster, which means there should be a way for someone who has thought about it to silence it. If that mechanism is generalized (e.g., some kind of configurable "join provider"), then cluster bootstrap could use that, rather than having Akka Cluster have to know about internal cluster bootstrap configuration details.

Could also be helpful to have the log message link to `https://doc.akka.io/docs/akka/$akkaVersion/cluster-usage.html#joining-to-seed-nodes`

Reply:
Added the link to docs in the log message, but adding a configurable join provider seems like overkill and haven't been requested for other other reasons. Will not do that in this PR, at least.